apps/bfd-server/bfd-server-launcher/src/main/assembly/bin/bfd-server-launcher.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+scriptDirectory="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+bfdServerLauncherJar="${scriptDirectory}/$(ls ${scriptDirectory} | grep '^bfd-server-launcher-.*jar')"
+classpath="${bfdServerLauncherJar}:${scriptDirectory}/lib/*"
+mainClass="gov.cms.bfd.server.launcher.DataServerLauncherApp"
+[ -n "${BFD_JAVA_HOME}" ] && javaExecutable=${BFD_JAVA_HOME}/bin/java || javaExecutable=java
+
+exec ${javaExecutable} -cp ${classpath} $* ${mainClass}
```

Comment:
Same note about `"$@"` being generally better in case an arg ever has a space in it.